### PR TITLE
[misc] Simplified README.md

### PR DESCRIPTION
**Pull request by yuanming-hu**
Related issue = #740 

More visual demos, and an introduction to the team members will be added later.

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,24 @@
...
 
 ```bash
 python3 -m pip install taichi
 ```
-**Supported environments:**
- - OS: Windows, Linux, and OS X
- - Python: 3.6/3.7/3.8
- - Backends: x64 CPUs/CUDA/Apple Metal.
- - Build from source if you work in other environments (e.g., you need the OpenGL backend or your CPU is ARM).
+**Supported OS**: Windows, Linux, and OS X; **Python**: 3.6/3.7/3.8; **Backends**: x64 CPUs/CUDA/Apple Metal.
```
Comment (archibate) [inline README.md:11]: Backends: OpenGL*

File: README.md [hunk 2]
```diff
@@ -1,97 +1,24 @@
...
-  <h3> <a href="https://taichi.readthedocs.io/en/latest/"> Documentation </a> | <a href="https://taichi.readthedocs.io/en/latest/hello.html"> Tutorial </a> | <a href="https://github.com/yuanming-hu/difftaichi"> DiffTaichi </a> | <a href="https://github.com/yuanming-hu/taichi/tree/master/examples"> Examples </a> | <a href="https://taichi.readthedocs.io/en/latest/contributor_guide.html"> Contribute </a> | <a href="https://forum.taichi.graphics/"> Forum </a> </h3>
+  <h3> <a href="https://taichi.readthedocs.io/en/latest/"> Documentation </a> | <a href="https://taichi.readthedocs.io/en/latest/hello.html"> Tutorial </a> | <a href="https://github.com/yuanming-hu/difftaichi"> DiffTaichi </a> | <a href="https://github.com/yuanming-hu/taichi/tree/master/examples"> Examples </a> | <a href="https://taichi.readthedocs.io/en/latest/contributor_guide.html"> Contributor Guidelines </a> | <a href="https://forum.taichi.graphics/"> Forum </a> </h3>
 </div>
 
-## Install [![Downloads](https://pepy.tech/badge/taichi-nightly)](https://pepy.tech/project/taichi-nightly)
+### Installation [![Downloads](https://pepy.tech/badge/taichi-nightly)](https://pepy.tech/project/taichi-nightly)
```
Comment (archibate) [inline README.md:6]: No nightly now?

File: README.md [hunk 3]
```diff
@@ -1,97 +1,24 @@
...
 
 ```bash
 python3 -m pip install taichi
 ```
-**Supported environments:**
- - OS: Windows, Linux, and OS X
- - Python: 3.6/3.7/3.8
- - Backends: x64 CPUs/CUDA/Apple Metal.
- - Build from source if you work in other environments (e.g., you need the OpenGL backend or your CPU is ARM).
```
Comment (archibate) [inline README.md:15]: I thought the previous list was clearer? **build from src provides other option**.

File: README.md (hunk 1 above)
Comment (yuanming-hu) [reply README.md:11]: Shall we leave v0.6 for officially announcing OpenGL? :-)

File: README.md (hunk 2 above)
Comment (yuanming-hu) [reply README.md:6]: No, simply `taichi` is easier for users.

File: README.md (hunk 1 above)
Comment (archibate) [reply README.md:11]: here we can tell them what they can got if build from src. e.g. cuda102, arm.

File: README.md (hunk 2 above)
Comment (archibate) [reply README.md:6]: this why is the link still taichi-nightly?

Comment (yuanming-hu) [reply README.md:6]: Because it's good to show how many people have downloaded `taichi-nightly` until the #downloads of `taichi` catches up.

**Pull request merged**